certbot/docs/api/certbot.compat.os.rst

```diff
@@ -2,6 +2,4 @@ certbot.compat.os module
 ========================
 
 .. automodule:: certbot.compat.os
-    :members:
-    :undoc-members:
-    :show-inheritance:
+    :members: chmod, umask, chown, open, mkdir, makedirs, rename, replace, access, stat, fstat
```

Comment:
Could you add a line in the `certbot.compat.os` to indicate that if a new function is added to it, then the RST documentation file needs to be update as well?